src/main/java/com/challenge/api/controller/auth/request/KakaoLoginRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.challenge.api.controller.auth.request;
+
+import com.challenge.api.service.auth.request.KakaoLoginServiceRequest;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+public class KakaoLoginRequest {
+
+    @NotBlank
+    private String accessToken;
+
+    public KakaoLoginServiceRequest toServiceRequest() {
+        return KakaoLoginServiceRequest.builder()
+                .accessToken(this.accessToken)
+                .build();
```

Comment:
```suggestion
    public KakaoLoginServiceRequest toServiceRequest() {
        return KakaoLoginServiceRequest.builder()
                .accessToken(accessToken)
                .build();
```
this.accessToken을 붙여서 명확하게 하는것도 좋은데 겹치는 필드가 아니기 때문에 코드 가독성을 위해 없애는게 더 좋다고 생각합니다 ~ 왜냐면 .builder().accessToken은 KakaoLoginServiceRequest를 바라 보고 있기 때문에 겹치는 필드가 아니고 IDE에서도 색상으로 잘 표시해주기 때문에 this 키워드가 없어도 충분히 의도가 명확히 전달되는것 같아요 ~

Reply:
아하 그럼 this 모두 제거하겠습니다!